templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car</h1>
+  <p>Are you sure you want to delete car: {{ car.model }}?</p>
+  <p><i>You can't revert this change!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Confirm" class="btn-danger">
```

Comment:
it will be a good option for the user to cancel deletion, that’s why we have an extra step before deleting something

Reply:
Added cancel button